users/views_test.py

```diff
@@ -46,7 +46,8 @@ def test_get_user_by_id(user_client, user):
 
 
 @pytest.mark.parametrize(
```

Comment:
Nit:

Can simplify to this, pytest will generate the permutations:
```python
@pytest.mark.parametrize("is_anonymous", [True, False])
@pytest.mark.parametrize("show_enrollment_codes", [True, False])
```
```

Reply:
Thanks, Updated.